var/spack/repos/builtin/packages/py-dm-haiku/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyDmHaiku(PythonPackage):
+    """JAX-based neural network library"""
+
+    homepage = "https://github.com/deepmind/dm-haiku"
+    url = "https://pypi.io/packages/source/d/dm-haiku/dm-haiku-0.0.5.tar.gz"
+
+    version('0.0.5', sha256='e986237e1f840aa3bd26217ecad84b611bf1456e2139f0f79ea71f9c6222d231')
+    depends_on('py-absl-py@0.7.1:', type=('build', 'run'))
+    depends_on('py-jmp@0.0.2:', type=('build', 'run'))
+    depends_on('py-numpy@1.18.0:', type=('build', 'run'))
+    depends_on('py-tabulate@0.8.9:', type=('build', 'run'))
+    depends_on('py-typing-extensions', type=('build', 'run'))
+    depends_on('py-jax', type=('build', 'run'))
```

Comment:
I don't see this dep listed anywhere

Reply:
from README.md:
```
    "Because JAX installation is different depending on your CUDA version, Haiku does
     not list JAX as a dependency in `requirements.txt`."
```